engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -173,18 +177,17 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId,
     case Event(RestartExecutingWorkflowCommand, _) =>
       // TODO: Restart executing
       goto(WorkflowExecutionInProgressState)
-    case Event(AbortExecutingWorkflowCommand, _) =>
-      context.parent ! WorkflowExecutionAbortedResponse
-      goto(WorkflowExecutionAbortedState)
   }
 
   when(WorkflowExecutionInProgressState) {
     case Event(BackendJobPreparationSucceeded(jobDescriptor, actorProps), stateData) =>
       pushPreparedJobMetadata(jobDescriptor.key, jobDescriptor.inputs)
-      context.actorOf(actorProps, buildJobExecutionActorName(jobDescriptor)) ! ExecuteJobCommand
+      val backendExecutionActor = context.actorOf(actorProps, buildJobExecutionActorName(jobDescriptor))
+      backendExecutionActor ! ExecuteJobCommand
       stay() using mergeExecutionDiff(stateData, WorkflowExecutionDiff(Map(jobDescriptor.key -> ExecutionStatus.Running)))
+        .addBackendJobExecutionActor(jobDescriptor.key, backendExecutionActor)
     case Event(BackendJobPreparationFailed(jobKey, t), stateData) =>
-      log.error(s"Failed to start job $jobKey", t)
+      log.error(s"Failed to start job $jobKey: ${t.getMessage}")
```

Comment:
ToL, for Akka Logging, we should eventually switch to using "[{}](http://doc.akka.io/docs/akka/current/java/logging.html)".